### PR TITLE
Added custom form with basic validation

### PR DESCRIPTION
**Pull request by dwshore**
Added form with specific fields and verification that form is filled out. Still need to add verification that the FV PW field meets the minimum requirements.

### CODE REVIEW

File: SplashBuddy/Views/Main/MainViewController.swift [hunk 1]
```diff
@@ -234,10 +235,62 @@ class MainViewController: NSViewController, NSTableViewDataSource {
         }
     }
 
-    @IBOutlet weak var sendButton: NSButton!
+    func errorReadingResults(missingData: String){
+        let alert = NSAlert()
+        alert.messageText = "Missing Data"
```
Comment (Ethenyl) [inline SplashBuddy/Views/Main/MainViewController.swift]: Please rely on localisable strings to display information to the user. Add two new keys to the different `Localizable.strings` files (one for each language).

Once everything is merged, I'll take care of adding the new strings to the online tool we're using to translate SB.

File: SplashBuddy/Views/Main/MainViewController.swift [hunk 2]
```diff
@@ -134,7 +134,7 @@ class MainViewController: NSViewController, NSTableViewDataSource {
     """
 
     internal func formEnterKey() {
-        self.evalForm(self.sendButton)
+        self.evalForm(self.sendButton2)
```
Comment (Ethenyl) [inline SplashBuddy/Views/Main/MainViewController.swift]: Is `sendButton` not used anymore?

**Comment by dwshore** [comment]
Thank you for blocking this merge. I intended to put this on my fork and it went to the primary instead. (Still learning, sorry about that!)

**Comment by Ethenyl** [comment]
@dwshore it's not a problem. If it's that simple, just retarget your PR toward your own project or close this one and recreate a new one.

**Pull request closed without merging**